editor/data-binding/lists.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: 'Lists'
+description: 'Use DataBinding to generate lists at runtime'
+---
+
+<Note>Lists are currently available in <a href="https://rive.app/blog/early-access-to-unreleased-features" target="_blank" rel="noopener noreferrer">Early Access</a> as a feature in the Rive Editor. Further performance enhancements as well as Runtime APIs will be available when released to the general public.</Note>
+
+A List is a way to display a set of items that are dynamically generated based on bound data values that you set up in your View Models. This allows you to build Rive files that can change in real time based on updates to those values. You can use Lists to create:
+
+- Menus with a dynamic amount of options
+- Product listings
+- Notifications or activity feeds
+- Chat messages
+- Dropdown menus
+- Contacts, friends, or followers lists
+- High scores, tables, and more
+
+---
+
+## Artboard Lists
+
+Artboard Lists enable you to generate a number of list items using an Artboard to represent each item in your List. Artboard Lists must be added as children of an Artboard or Layout. 
+
+To add an Artboard List to the stage, first select either an Artboard or a Layout. In the inspector on the right side of the editor, you will see a button to add an Artboard List. Click it to add the List to your hierarchy. It will appear as a child of the Artboard or Layout had previously selected.
+
+![Image](/images/editor/add-artboard-list.png)
+
+Once the Artboard List is added to your hierarchy, you can select it and see its inspector.
+
+![Image](/images/editor/artboard-list-property.png)
+
+The next step is to bind data to it using DataBinding. This will determine the content and number of items in your List. There are 2 ways to generate List content:
```

Comment:
```suggestion
The next step is to bind data to it using Data Binding. This will determine the content and number of items in your List. There are 2 ways to generate List content:
```